### PR TITLE
Generate documentation for Harmonic

### PR DESCRIPTION
**Pull request by azeey**
Updates the infrastructure to build and upload docs for Harmonic.

### CODE REVIEW

**Review by iche033 — APPROVED** [review]

**Review by mjcarroll — APPROVED** [review]

**Review by caguero — APPROVED** [review]

**Comment by azeey** [comment]
Oops, forgot a file. PTAL one more time

File: tools/Dockerfile.harmonic [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+COPY scripts/install_common_deps.sh scripts/install_common_deps.sh
+RUN scripts/install_common_deps.sh $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY
+RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-prerelease $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/gazebo-prerelease.list > /dev/null \
+  apt-get update
+
+COPY scripts/build_gz.sh scripts/build_gz.sh
+
+RUN echo ::endgroup::
+
+RUN scripts/build_gz.sh gazebosim gz-cmake gz-cmake3 n \
```
[the remark this answers is not in the document]
Comment (azeey) [reply tools/Dockerfile.harmonic:26]: Coped from Dockerfile.garden, but I'm guessing they tell the script if the repo has code or not.

[the remark this answers is not in the document]
Comment (azeey) [reply tools/Dockerfile.harmonic:26]: Ah https://github.com/azeey/docs/blob/698c7e96d20577061728bc791bef56d6d453d42f/tools/scripts/build_gz.sh#L6

But I'm not sure why gz-cmake3 is `n`. It does have some tutorials.

[the remark this answers is not in the document]
Comment (azeey) [reply tools/Dockerfile.harmonic]: Changed it to `y` in https://github.com/gazebosim/docs/pull/378/commits/1692d8068c42a11a95e51e5e250dfb45dd67f0e1 and also added `gz-utils`.

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**